lib/utils.js

```diff
@@ -647,8 +647,6 @@ exports.stackTraceFilter = function() {
   function isMochaInternal(line) {
     return (
       ~line.indexOf('node_modules' + slash + 'mocha' + slash) ||
-      ~line.indexOf('node_modules' + slash + 'mocha.js') ||
```

Comment:
Why are these lines removed? As git history, they are related with [windows issues](https://github.com/mochajs/mocha/commit/8bf1f297142b02128664523ab5a9b3d55b09c365).

Reply:
they are extraneous. the next line catches them:

```js
(~line.indexOf(slash + 'mocha.js'))
```

the windows-specific changes have to do with the value of `slash`.